langchain_tools_demo/main.py

```diff
@@ -32,10 +33,31 @@
     messages_to_dict,
 )
 from markdown import markdown
+from piny import StrictMatcher, YamlLoader  # type: ignore
+from pydantic import BaseModel
 from starlette.middleware.sessions import SessionMiddleware
 
 from agent import init_agent, user_agents
 
+BASE_HISTORY: list[BaseMessage] = [
+    AIMessage(content="I am an SFO Airport Assistant, ready to assist you.")
+]
+routes = APIRouter()
+
+
+class AppConfig(BaseModel):
+    host: IPv4Address | IPv6Address = IPv4Address("0.0.0.0")
```

Comment:
nit: should we default to 127.0.0.1?

Reply:
Cloud build will fail with 127.0.0.1. Will keep it as 0.0.0.0